tests/functional/test_completion.py

```diff
@@ -107,9 +107,10 @@ def test_completion_alone(autocomplete_script):
     """
     Test getting completion for none shell, just pip completion
     """
-    result = autocomplete_script.pip('completion', allow_stderr_error=True)
+    result = autocomplete_script.pip('completion', expect_error=True)
```

Comment:
Just to know, what is the difference in behavior of these two?

Reply:
I am not sure about the difference conceptually. 

I am returning `status_code` instead of None in the `run` method of completion, and in the other examples I saw where we were asserting returncode, I saw `expect_error` being used